node/runtime/src/lib.rs

```diff
@@ -169,7 +170,7 @@ impl treasury::Trait for Runtime {
 }
 
 impl contract::Trait for Runtime {
-	type Currency = balances::Module<Runtime>;
+	type Currency = Balances;
```

Comment:
I think `balances::Module<Self>` and `balances::Module<Runtime>` and `Balances` are same. But better to be consistent.